core/src/main/java/com/beepiz/bluetooth/gattcoroutines/experimental/GattConnectionImpl.kt

```diff
@@ -121,8 +111,17 @@ internal class GattConnectionImpl(
         gatt.discoverServices()
     }
 
-    override fun setCharacteristicNotificationsEnabled(characteristic: BGC, enable: Boolean) {
+    override fun setCharacteristicNotificationsEnabled(characteristic: BGC, enable: Boolean, type: ByteArray) {
         gatt.setCharacteristicNotification(characteristic, enable).checkOperationInitiationSucceeded()
+
+
+        val descriptor = characteristic.getDescriptor(descriptorUUID)
+        when(enable) {
```

Comment:
`when` is a keyword, not a function call. As a result, there should be a space between it and the opening parenthesis.
Reformatting from the IDE (cmd/ctrl + option/alt + enter) should fix that automatically BTW.

Reply:
👍